src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+    private static final String SPACE = "";
+    private static final String SPECIAL_CHARACTERS = "[,\\s\\-:\\?]";
+    private static final String ONLY_WORD_PATTERN = "[^a-z A-Z]";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
let's try to simplify this solution - you don't need 3 loops, now for inside of while is enough. 
Inside while split your line with correct regex and and append lower cased version to the result.
After that you will just need to sort your words.